docs/cluster-deployment/stripe.rst

```diff
@@ -0,0 +1,164 @@
+   :alt: CrateDB Cloud configuration wizard step 1
+
+Click *Next* to proceed.
+
+
+Wizard step 2
+-------------
+
+In the next step, we come to the cluster itself. Here the wizard will ask you
+to name the cluster, as well as the username and the password that will
+subsequently be used to access the cluster via the unique cluster URL. The
+password must be at least 24 characters long; any characters are accepted,
+including special characters. If you want, click the *Auto-generate password*
+button to automatically generate a secure 24 character password (if it shows
+a password already, clicking again will generate a new one).
+
+.. WARNING::
+
+    The password will not be retrievable past this point. You may want to copy
+    the password from the password field into a secure location (e.g., a
+    password manager).
+
+.. image:: ../_assets/img/stripe-wizard-step2.png
+   :alt: CrateDB Cloud configuration wizard step 2
+
+Finally, you can also set the scale unit of the cluster to the desired level
+here. As you move the slider horizontally, you will move up (or down) the scale
+levels within the subscription plan you previously selected. As you will see,
+the hardware capacities of the cluster will change correspondingly. Currently,
+within each subscription plan clusters can be scaled between scale units 1-3.
+The default scale unit is 1. Note that scaling the cluster changes its price.
+
+Do not worry, however: clusters can be scaled up or down as needed - for
+example if your use case changes - at any point later on. To understand more
+about subscription plans, scaling, and scale units, refer to our `reference on
+subscription plans`_ and our `scaling guide`_.
+
+When the names and password are generated, click *Next* to continue.
+
+
+Wizard step 3
+-------------
+
+This step is even easier than the others: it merely summarizes the results
+of your choices. First, it shows the settings for your organization and
+project, with the names you have defined. Next, it shows the cluster
+information, including the version of CrateDB the cluster will be running and
+once again the scale unit capacities the cluster will have. Finally, the
+pricing information shows you the relevant costs of running the cluster. Note
+that Crate.io always bills for usage on an hourly basis, and only actual usage
+is ever billed.
+
+.. image:: ../_assets/img/stripe-wizard-step3.png
+   :alt: CrateDB Cloud configuration wizard step 3
+
+As always, simply click *Next* to proceed.
+
+
+Wizard step 4
+-------------
+
+One final step remains, however. At this stage, the wizard will prompt you for
+your credit card information to bill for the cluster. Fill out the number,
+expiry date, and CVC (the three numbers on the back of the card) for the credit
+card you intend to use to pay for the CrateDB Cloud cluster. Additionally,
+provide your billing address. Please do not forget to tick the box to authorize
+Crate.io to take payments from your card.
```

Comment:
Maybe one for later, but I think we need a section on how the billing is going to work (i.e. where will the invoices arrive etc.)

Reply:
Yeah good point! Don't think a deployment tutorial is the place to do it, though. But I'll write up a proposal about it.